models/Asset/Service.php

```diff
@@ -336,37 +354,46 @@ public static function rewriteIds($asset, $rewriteConfig)
         return $asset;
     }
 
+    public static function getListfolderGridData($item) {
```

Comment:
please add phphdoc and maybe a short description too

Reply:
dead code, removed it